src/Microsoft.DotNet.Darc/src/DarcLib/Helpers/GitFileManager.cs

```diff
@@ -1042,6 +1112,58 @@ private Task<bool> VerifyMatchingVersionProps(IEnumerable<DependencyDetail> depe
             return Task.FromResult(result);
         }
 
+        /// <summary>
+        ///     Verify that any dependency that exists in global.json and Version.Details.xml (e.g. Arcade SDK) 
```

Comment:
I'm confused, this summary mentions global.json and version.details.xml, is one of the files supposed to be dotnet-tools.json?

Reply:
I'll try to rewrite it.